firmware/apps/nic/pv.uc

```diff
@@ -74,23 +74,27 @@ passert(PV_MAX_CLONES, "EQ", 2)
  *       +-+---+-----+-------------------+-----+-------------------------+
  *    10 |A|    0    |   Packet Number   |  0  |         Offset          | 2
  *       +-+---------+-------------------+-----+---------+---------------+
- *    11 |        Sequence Number        | --- | Seq Ctx |   Protocol    | 3
+ *    11 |        Sequence Number        | XML | Seq Ctx |   Protocol    | 3
  *       +-------------------------------+-+-+-+---------+---+-+-+-+-+-+-+
- *    12 |         TX Host Flags         |M|B|Seek (64B algn)|-|Q|I|i|C|c| 4
+ *    12 |         TX Host Flags         |M|B|Seek (64B algn)|X|Q|I|i|C|c| 4
  *       +-------------------------------+-+-+---------------+-+-+-+-+-+-+
  *    13 |       8B Header Offsets (stacked outermost to innermost)      | 5
  *       +-----------------+-----+-----------------------+---------------+
  *    14 |  Ingress Queue  | LMP |        VLAN ID        | Queue Offset  | 6
  *       +-----------------+-----+-----------------------+---------------+
  *    15 |                      Metadata Type Fields                     | 7
  *       +---------------------------------------------------------------+
+ * 16-23 |                      XDP Metadata                             | 0-7 <- ACTIVE_LM_ADDR_2 ()
+ *       +---------------------------------------------------------------+
```

Comment:
Do we _need_ to lock down LM addr2 for this? Are we sure it's not used elsewhere?

Reply:
I am allocating data by extending `PV_SIZE_LW` to 24 words. Shouldn't `pv_push` and `pv_pop` work as is?

Reply:
Maybe... It certainly looks like it from inspecting the code. (Note to self, write a unit test to confirm this)